xmppserver/src/main/java/org/jivesoftware/openfire/commands/event/GroupMemberAdded.java

```diff
@@ -71,37 +69,43 @@ public void execute(SessionData sessionData, Element command) {
             return;
         }
 
-        // Creates event params.
-        Map<String, Object> params = null;
-        
-        try {
+        final String wasAdminValue = get(data, "wasAdmin", 0);
+        if (wasAdminValue == null) {
+            note.addAttribute("type", "error");
+            note.setText("WasAdmin required parameter.");
+            return;
+        }
+        final boolean wasAdmin = "1".equals(wasAdminValue) || Boolean.parseBoolean(wasAdminValue);
```

Comment:
What's populating the "wasAdmin"?

Reply:
A new data field in the form that is passed into this method by the `data` variable.